src/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/output.css">
+  <title>Google</title>
+</head>
+
+<body class="font-Roboto text-secondary_dark flex flex-col h-screen">
+  <header class="flex justify-between items-center px-4">
+    <nav class="flex gap-4 items-center">
+      <svg class="w-6 fill-secondary_dark" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+        <path d="M0 0h24v24H0z" fill="none"></path>
+        <path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"></path>
+      </svg>
+      <a href="#" class="text-xs font-[600] py-4 px-2 text-primary_blue border-b-2 border-primary_blue">ALL</a>
```

Comment:
you should not use arbitrary values in tailwind , if needed add them in a configuration file